### PR TITLE
Change new line value from /n to <br> so the email is correctly formtted

### LINKED ISSUE #79: New lines between projects
Format email body such that it contains new lines between individual projects

### PR DESCRIPTION
**Pull request by xjusko**
resolves #79

### CODE REVIEW

File: src/main/java/org/jboss/set/draw/Lottery.java [hunk 1]
```diff
@@ -106,7 +106,7 @@ public static String createEmailText(String email, List<Issue> issues) {
                                 projectLinksEntry.getValue().stream()
                                         .map(issue -> LINK_LINE.formatted(issue.getBrowseUri().toString()))
                                         .collect(Collectors.joining()))
-                        .collect(joining("\n")),
+                        .collect(joining("<br>")),
```
Comment (The-Huginn) [inline src/main/java/org/jboss/set/draw/Lottery.java:109]:
```suggestion
                        .collect(joining("<br>\n")),
```

So the string represantation is still human-readable

**Review by The-Huginn — CHANGES_REQUESTED** [review]
I think we can also update itemization of issues utilizing unordered list <ul> html tag, which also adds the bullet character. Please see https://www.w3schools.com/tags/tag_li.asp
Also, I might be worth it adding <br> tag into the `PROJECT_LINE`.Thus would be `PROJECT_LINE = "...<br>\n" so it still looks readable in String form.

File: src/main/java/org/jboss/set/draw/Lottery.java [hunk 2]
```diff
@@ -98,16 +92,21 @@ public static String getUsername(String email) {
...
+
+                    String issueLinks = projectIssues.stream()
+                            .map(issue -> LINK_LINE.formatted(issue.getBrowseUri().toString()))
+                            .collect(Collectors.joining("\n"));
+
+                    return PROJECT_LINE.formatted(projectName) + "<ul>" + issueLinks + "</ul>";
+                })
+                .collect(Collectors.joining("\n"));
+
+        return String.format(EMAIL_BODY, getUsername(email), projectHtml, configFileUrl());
```
Comment (The-Huginn) [inline src/main/java/org/jboss/set/draw/Lottery.java:109]:
```suggestion
        return EMAIL_BODY.formatted(getUsername(email), projectHtml, configFileUrl());
```

**Review by The-Huginn — APPROVED** [review]
Except the suggestion, LGTM

**Comment by xjusko** [comment]
For the reference, this is how the email will be formatted after the changes.
image

**Pull request merged**